app/Locale/French.php

```diff
@@ -26,25 +26,38 @@ public function get_locale(): array
 			'DEFAULT' => 'Valeur par defaut',
 			'ALBUM_SET_ORDER' => 'Changer l\'ordre',
 			'ALBUM_ORDERING' => 'Trier par',
+			'ALBUM_OWNER' => 'Owner',
```

Comment:
```suggestion
			'ALBUM_OWNER' => 'Propriétaire',
```

Reply:
Thisis actually from a commit in master that is not yet in `consisten_json_api`. Should I apply this anyway?

Reply:
No, please not. This should be in another PR. All these should be added to https://github.com/LycheeOrg/Lychee/pull/1271.